Makefile

```diff
@@ -57,6 +58,13 @@ check-docker:
 	  exit 2; \
 	fi
 
+.PHONY: check-docker-experimental
+check-docker-experimental:
+	@if [ $$(docker version -f '{{json .Server.Experimental}}') != true ]; then \
+	  echo "Docker experimental needs to be enable"; \
```

Comment:
typo: "enable" should be "enabled".

Reply:
I have removed the target: check-docker-experimental

Reply:
I have removed the target: check-docker-experimental.
Now, cross-building does not depend on any demo version of docker.